src/lib/types/minions.ts

```diff
@@ -401,6 +401,10 @@ export interface TitheFarmActivityTaskOptions extends MinigameActivityTaskOption
 export interface SepulchreActivityTaskOptions extends MinigameActivityTaskOptions {
 	type: 'Sepulchre';
 	floors: number[];
+	fletch?: {
+		fletchableName: string;
+		fletchingQuantity: number;
+	};
```

Comment:
Just a little thing, but to make the stored json a bit smaller, It would be preferable to rename these to 
```ts
	fletch?: {
		id: number;
		qty: number;
	};
```